src/platform/iotjs_module_gpio-linux-general.inl.h

```diff
@@ -78,139 +62,123 @@ bool SetPinDirection(int32_t pin, GpioDirection direction) {
     strcpy(buffer, "out");
   }
 
```

Comment:
What about using following code?

    const char* buffer = (direction == kGpioDirectionIn) ? "in" : "out";